docs/sql/statements/explain.rst

```diff
@@ -52,6 +52,14 @@ The output of ``EXPLAIN ANALYZE`` also includes a break down of the query
 execution if the statement being explained involves queries which are executed
 using Lucene.
 
+.. NOTE::
+
+   When a query involves an empty partitioned table you may be surprised to
+   see no breakdown concerning that table, this is because there is only a
+   template and no actual table until at least one partition is created
```

Comment:
I would skip explaining the template as it's internal, I'd only state that you need at least on partition .